src/utils.js

```diff
@@ -0,0 +1,110 @@
+export const Shuffle = (array) =>{
+    for (var k = 0; k < array.length; k++) {
+        var i = array[k].length;
+        if (i === 0)
+            return false;
+        else {
+            while (--i) {
+                var j = Math.floor(Math.random() * (i + 1));
+                var tempi = array[k][i];
+                var tempj = array[k][j];
+                array[k][i] = tempj;
+                array[k][j] = tempi;
+            }
+        }
+    }
+    return array
+}
+
+export const rando = () =>{
+        const colors = ['purple', 'orange', 'red', 'green', 'yellow', 'blue', 'white', 'blue']
+        let randomColor = colors[Math.floor(Math.random()*colors.length)];
```

Comment:
Using the length of the colors array here is a good example of the opposite of a magic constant problem -- you can add and remove colors from the list without having to update this line.